quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/ProxyHostnameV2DistTest.java

```diff
@@ -55,6 +57,19 @@ public void testSchemeAndPortFromRequestWhenNoProxySet() {
         assertForwardedHeaderIsIgnored();
         assertXForwardedHeadersAreIgnored();
     }
+    
+    @Test
+    void testTrustedProxiesWithoutProxyHeaders(KeycloakDistribution distribution) {
+        CLIResult result = distribution.run("start-dev", "--proxy-trusted-addresses=1.0.0.0:8084");
```

Comment:
It's valid according to: https://quarkus.io/guides/all-config#quarkus-vertx-http_quarkus-http-proxy-trusted-proxies

Reply:
Thank you for pointing me at the docs. It seems that Quarkus actually implements it this way, still the documentation of the property in Keycloak is then incomplete as it doesn't point out that port numbers can be used with the trusted proxy address. Reading the Quarkus property again, this is then about "socket addresses", where I originally thought only host addresses would be specified.

Thinking about this a bit deeper, I think this is a strange configuration to also provide the port number: A TCP connection is identified by the the four elements source IP, source port, target port and target IP. The target port and target IP are where Quarkus listens for incoming connections. Source host is the proxy. If I also limit the source port, the proxy I configure here can only establish a single connection at a time. And even if that connection is closed, it might take a while until it can open a new connection while it is still in CLOSE_WAIT AFAIK. I'll raise a question with Quarkus, still waiting for the answer is out of scope for this PR as it is already implemented that way in Quarkus, and people might actually use it. 

I'll suggest a re-working in a minute.

Reply:
Yes, I left it out of the description - it's already getting pretty long, I'm fine with making it longer as it seems that pointing directly to the quarkus docs would be frowned upon. As for what is in the tests, I just need a placeholder value that in the one case will cause the headers to be ignored. Those can of course be changed if you want.

Reply:
Let's leave the placeholder as is, works for me. See below for one more sentence :-(

Reply:
Looks good

Reply:
https://github.com/quarkusio/quarkus/issues/42760